src/scss/layouts/_modalPopup.scss

```diff
@@ -0,0 +1,19 @@
+.modalPopup {
+  display: block;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%) scale(1);
+  transition: 0.2s ease-in-out;
+  height: 600px;
+  width: 700px;
+  min-height: 500px;
+  background-color: #fff;
+  border: 1px solid red;
+  border-radius: 10px;
+  z-index: 10;
+}
+
+.modalPopup.active {
```

Comment:
Spróbuj wykorzystać scss i BEM

Reply:
Jeszcze pracuje nad kodem, wrzucone, żeby Jarosław mógł zacząć swojego taska

Reply:
Daj znać jak skończysz pracę bo ja za każdym raziem jak wchodzę w tego PR, to nie robie Review tylko dlatego, że jeszcze jest w trakcie robienia :D